go/pkg/apis/enricher/php_enricher.go

```diff
@@ -0,0 +1,99 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ * Red Hat, Inc.
+ ******************************************************************************/
+
+package enricher
+
+import (
+	"context"
+	framework "github.com/redhat-developer/alizer/go/pkg/apis/enricher/framework/php"
+	"github.com/redhat-developer/alizer/go/pkg/apis/model"
+	"github.com/redhat-developer/alizer/go/pkg/utils"
+	langfile "github.com/redhat-developer/alizer/go/pkg/utils/langfiles"
+)
+
+type PHPEnricher struct{}
+
+func getPHPFrameworkDetectors() []FrameworkDetectorWithConfigFile {
+	return []FrameworkDetectorWithConfigFile{
+		&framework.LaravelDetector{},
+	}
+}
+
+func (p PHPEnricher) GetSupportedLanguages() []string {
+	return []string{"php"}
+}
+
+// DoEnrichLanguage runs DoFrameworkDetection with found php project files.
+// php project files: composer.json
+func (p PHPEnricher) DoEnrichLanguage(language *model.Language, files *[]string) {
+	composerJson := utils.GetFile(files, "composer.json")
+
+	if composerJson != "" {
+		var targetLanguage string
+		if utils.IsTagInPackageJsonFile(composerJson, "php") {
```

Comment:
The `IsTagInPackageJsonFile` looks for the tag inside dependencies. The `composer.json` file, afaik it doesn't include a section of dependencies. So in case another language was detected, and we need to create a PHP component, the PHP language will not be detected.

Reply:
Good catch! Updated result:

```shell
➜  alizer-testing ./alizer component devfile-stack-php-laravel
[
	{
		"Name": "devfile-stack-php-laravel",
		"Path": "devfile-stack-php-laravel/",
		"Languages": [
			{
				"Name": "PHP",
				"Aliases": [
					"inc"
				],
				"Weight": 94,
				"Frameworks": [
					"Laravel"
				],
				"Tools": [],
				"CanBeComponent": true
			},
			{
				"Name": "Hack",
				"Aliases": null,
				"Weight": 94,
				"Frameworks": [],
				"Tools": [],
				"CanBeComponent": false
			},
			{
				"Name": "JavaScript",
				"Aliases": [
					"js",
					"node",
					"nodejs",
					"TypeScript"
				],
				"Weight": 5,
				"Frameworks": [],
				"Tools": [
					"NodeJs",
					"Node.js"
				],
				"CanBeComponent": true
			}
		],
		"Ports": null
	}
]
```